frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -407,14 +400,31 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
     this.props.onChangeUser("gpuMemoryFactor", gpuFactor);
   };
 
-  handleFindData = async (layerName: string) => {
+  handleFindData = async (layerName: string, isColorLayer: boolean) => {
+    const { volume, tracingStore } = Store.getState().tracing;
     const { dataset } = this.props;
-    const { position, resolution } = await findDataPositionForLayer(
-      dataset.dataStore.url,
-      dataset,
-      layerName,
-    );
-    if (!position || !resolution) {
+    let foundPosition;
+    let foundResolution;
+
+    if (volume && !isColorLayer) {
+      const requestUrl = `${tracingStore.url}/tracings/volume/${volume.tracingId}`;
+      const { position, resolution } = await findDataPositionForLayer(requestUrl);
+      if ((!position || !resolution) && volume.fallbackLayer) {
+        await this.handleFindData(volume.fallbackLayer, true);
```

Comment:
This feels a bit hacky, since you are passing `isColorLayer=true` here, even though this call is meant for the fallback layer of the segmentation. Can you restructure this code to avoid this?

Reply:
There's probably a better way, but renaming `isColorLayer` to `isDataLayer` might work here (isDataLayer being false means that the volume tracing should be looked up).